backend/tests/integration/test_mcp_summarization_with_event_id.py

```
@@ -0,0 +1,271 @@
+    LLMMessage,
+    MessageRole,
+)
+
+
+@pytest.mark.integration
+class TestMCPSummarizationWithEventID:
+    """Test end-to-end summarization with mcp_event_id tracking."""
+    
+    @pytest.mark.asyncio
+    async def test_summarization_persists_mcp_event_id_to_database(
+        self, test_database_session: Session
```

Ruff (0.14.1)

Unused method argument: `test_database_session`

(ARG002)